### ISSUE TITLE
PADing error on resnet model from Pytorch

### ISSUE THREAD

**Issue by inakinavarro** [onnx/onnx-tensorflow#188]
When converting a model ResNet based model created with Pytorch I get the following warning message:

_onnx_tf/backends/backend_v1.py:123: UserWarning: Using the pooling op in compatibility mode.This means your graph cannot be serialized.Please configure your pooling operation to only use paddings that correspond to Tensorflow SAME or VALID padding.
  "correspond to Tensorflow SAME or VALID padding.", UserWarning)_

I use the following code to convert:
python onnx_2_tf.py hope.onnx

```
import onnx_tf.backend as tf_backend
import onnx
import argparse

def main():
    parser = argparse.ArgumentParser()
    parser.add_argument('model', help='onnx file')
    args = parser.parse_args()

    model = onnx.load(args.model)
    tf_backend.prepare(model)

if __name__ == "__main__":
    main()
```
Here you can find the model:
https://www.dropbox.com/s/nj2xceg14ctwdxr/hope.onnx?dl=0

**Python, ONNX, ONNX-TF, Tensorflow version**

This section can be obtained by running `get_version.py` from util folder.
 - Python version: 2.7 
 - ONNX version: master
 - ONNX-TF version: onnx-tf==1.1.1
 - Tensorflow version: 1.6

**Comment by fumihwh** [comment]
Maybe duplicate of #167
